app/src/main/java/lt/vilnius/tvarkau/BaseActivity.java

```diff
@@ -17,7 +17,7 @@
     @Inject
     SharedPreferences myProblemsPreferences;
 
-    private ApplicationComponent component;
+    protected ApplicationComponent component;
```

Comment:
If access level was changed only for testing purposes add `VisibleForTesting` annotation. For instance: 
```java
@VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
```